model/coverage.py

```diff
@@ -100,6 +100,11 @@ class Timestamp(Base):
     COVERAGE_PROVIDER_TYPE = "coverage_provider"
     SCRIPT_TYPE = "script"
 
+    # A stand-in value used to indicate that a field in the timestamps
+    # should be set to None. This is necessary because 'None' generally
+    # means 'use the default value'.
+    NO_VALUE = object()
```

Comment:
Why/when would you want a Timestamp.finish to be able to take on a null value?

Reply:
In most cases it's not necessary. The motivating case is `MWCollectionUpdateMonitor` in circulation, which uses `timestamps.finish` to ask for changes since a certain time. If that monitor gets an empty OPDS feed, we want to leave `timestamps.finish` alone, not update it to the current time.

This is to avoid the possibility that there are pending changes that didn't show up the last time we asked for changes since time X, but will show up the next time we ask.

In the case where `timestamps.finish` starts out None, we want to make sure it stays None.

The BibliothecaEventMonitor used to be the same way -- it used `timestamps.start` and `timestamps.finish` to track the timestamps of the first and last events it saw in a given run. However, that was only for informational purposes, and it wasn't really that interesting, so I was able to convert that monitor over to a TimelineMonitor.

(It might not be that bad to also convert MWCollectionUpdateMonitor, but missing an update from the metadata wrangler is worse than missing one from Bibliotheca, because we have no metadata wrangler equivalent of the BibliothecaCirculationSweep.)